x-pack/filebeat/input/s3/input.go

```diff
@@ -251,14 +252,16 @@ func (p *s3Input) processMessage(svcS3 s3iface.ClientAPI, message sqs.Message, w
 		p.logger.Error(errors.Wrap(err, "handleSQSMessage failed"))
 		return
 	}
+	p.logger.Debugf("handleSQSMessage succeed and returned %v sets of S3 log info", len(s3Infos))
```

Comment:
Do we want all these debug messages in? Other than that, LGTM.

Reply:
Yeah, right now we don't have enough to check where the code went wrong 😂 These messages won't show up unless DEBUG logging is enabled.